cors-policy/pom.xml

```diff
@@ -24,9 +24,28 @@
       <artifactId>apiman-gateway-engine-policies</artifactId>
       <scope>provided</scope>
     </dependency>
+    
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>4.11</version>
```

Comment:
Can I extract these deps and versions into the parent pom, or do we want to keep each plugin independent? I wasn't sure what our strategy was going to be on this.